etc/helm/pachyderm/values.yaml

```diff
@@ -15,6 +15,10 @@ global:
     # When using autogenerated value for the initial install, it must be pulled from the
     # postgres secret and added to values.yaml for future helm upgrades
     postgresqlPassword: "" 
+    # If you want to supply the postgresql password in an existing secret, leave Password blank and
+    # Supply the name of the existing secret in the namespace and the key in that secret with the password
+    postgresqlExistingSecretName: ""
```

Comment:
Possibly a bit pedantic, but I think having `postgresql` in the value name is redundant, because the value path already contains it. `existingSecretName`/`existingSecretKey` makes more sense to me, but I get that you're following the established pattern here.

Reply:
agree but you've got it